azure-pipelines.sds.pr-release.yml

```diff
@@ -0,0 +1,144 @@
+resources:
+  repositories:
+    - repository: azTemplates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/master
+      endpoint: hmcts
+
+parameters:
+  - name: env
+    type: string
+    default: dev
+    values:
+      - dev
+
+  - name: git_repo_name
+    type: string
+    default: bookings-api
+
+stages:
+  - stage: Dock
+    displayName: Build and Push Docker Image
+    variables: 
+      - group: aks-cluster-info
+      - template: variables/shared.yaml
+        parameters:
+          env: ${{ parameters.env }}
+          git_repo_name: ${{ parameters.git_repo_name }}
+          cluster: $(cluster)
+      - group: vh-github-app-credentials
+      - name: prTagName
+        value: 'pr-$(System.PullRequest.PullRequestNumber)'
+
+    jobs:
+    - job:
+      steps:
+        - checkout: self
+
+        - template: templates\Containerisation\docker\docker-compose.yaml@azTemplates
+          parameters:
+            azureSubscription: ${{ variables.acrSubscription }}
+            acrName: ${{ variables.acrName }}
+            repositoryName:  ${{ variables.repositoryName }}
+
+        - template: templates\Containerisation\docker\docker-push-azure.yaml@azTemplates
+          parameters:
+            azureSubscription: ${{ variables.acrSubscription }}
+            acrName: ${{ variables.acrName }}
+            repositoryName:  ${{ variables.repositoryName }}
+            imageName: "${{ replace(parameters.git_repo_name, '-', '') }}"
+            imageTags:
+              - '${{ variables.prTagName }}'
```

Comment:
Do we need to push to acr if we are straight pushing the helm to AKS in deployment?

Reply:
Would it not be good to have a record of the image regardless? The ACR ain't gonna fill up

Reply:
OH, although if all interactions on dev will be via this pipeline maybe there's no point having an image in the ACR - sure how can it be deployed...